### PR TITLE
ported in my VFS

### PR DESCRIPTION
**Pull request by laurinyev**
I didnt hook it up cuz Im lazy to to write a test device, also I was suprised how little changes it needed
it may not work if your strcmp and strncmp is not spec compliant

### CODE REVIEW

File: VFS/vfs.cpp [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+    struct node* file = vfs_inspect(path);
+    return read_file(file->ext.ext_file.disk_id,(char*)file->ext.ext_file.contents_on_disk_path,offset,size);
+}
+void vfs_write(char* path,struct file_buffer buf,uint64_t offset){
+    struct node* file = vfs_inspect(path);
+    write_file(file->ext.ext_file.disk_id,(char*)file->ext.ext_file.contents_on_disk_path,offset,buf);
+}
+
+struct node *vfs_inspect(char *path) {
+    if (strcmp(path, "/") == 0) {
```
Comment (cre0z) [inline VFS/vfs.cpp:129]: `strcmp()` can be unsafe, using `strncmp()` is more desirable

Comment (laurinyev) [reply VFS/vfs.cpp:129]: so you want `/bin` be root?????
that is stupid

Comment (laurinyev) [reply VFS/vfs.cpp:129]: I thinki you dont understand what this check is for

Comment (SachaTending) [reply VFS/vfs.cpp:129]: wrong repo btw

Comment (laurinyev) [reply VFS/vfs.cpp:129]: > wrong repo btw

port it yourself then, my changes are here, YOU CAN DO IT

**Pull request closed without merging**